### DISCUSSION TITLE
Check details missing when "All checks have passed"

### DISCUSSION THREAD

**Discussion by mboos** [community/community#69844] [Actions]
### Select Topic Area

Bug

### Body

Previously, I was able to view details of checks on main branch commits that passed. Now however, when I click on a green checkmark next to a commit, I only see "All checks have passed" and the number of checks, but I don't see any links to the checks themselves like I do for failed or incomplete sets of checks. This makes diagnosing failures in subsequent commits harder because I can't as easily find passing logs to compare with failing ones.

**Comment by VerifiedJoseph** [comment]
Also having this issue. 

From looking at CSS, the HTML element containing the check details has its visibility set to `hidden`. Remove that property and it displays as expected.

<details>
<summary>Screenshot</summary>

![image](https://github.com/community/community/assets/20118140/206eca75-8d9c-4183-ac71-3a56dfd76a12)

</details>

**Reply by mboos** [reply]
Thanks, that's helpful to know. Very inconvenient to have to pull up the dev console to find the links though.

**Comment by tt-rkim** [comment]
This is a very strange change. Is there any reason why this is desirable?

Our team members would like to see what tests have run, as the list can change anytime and also it's nice to have direct access to the workflow runs via links in the checkmark.

**Comment by essenmitsosse** [comment]
There is also a regression where the popover is misplaced and sticks out of the window since today, so I suspect someone deployed style changes recently that break these popovers.

![image](https://github.com/community/community/assets/4087618/0c8f2432-641b-4541-a09b-a63ed963bec3)

**Reply by essenmitsosse** [reply]
This and the original problem seem to be fixed now.
